src/logger/logger.middleware.ts

```diff
@@ -0,0 +1,9 @@
+import { Injectable, NestMiddleware } from '@nestjs/common';
+
+@Injectable()
+export class LoggerMiddleware implements NestMiddleware {
+  use(req: any, res: any, next: () => void) {
```

Comment:
Owerall good job!
But please rename this LoggerMiddleware to LogIpMiddleware and create specific directory for it.
src/middlewares/log-ip

Reply:
done